verifyFiles.py

```diff
@@ -512,7 +512,7 @@ def main():
                                              /home/$(USER)/CentOS7.FSL5.0.6
                                              Each directory will contain subject folders like 100307,100308 etc'''))
         parser.add_argument("-c", "--checksumFile",help="Reads checksum from files. Doesn't compute checksums locally")
-	parser.add_argument("difference", help="Writes the difference matrices and indexs into files")
+	parser.add_argument("difference", help="Base name to use in output file names")
```

Comment:
Rename parameter `result_base_name`. Explain in the help that the following files will be written: <result_base_name>_....txt (small explanation), etc.